.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+ - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+   open-pull-requests-limit: 5   
```

Comment:
Would we want this limit? In a monthly schedule, several dependencies could have been upgraded.

IMO for external deps there should not be a limit

Reply:
done